packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -219,6 +222,8 @@ export default defineComponent({
       )
     }
 
+    const expirationRules = computed(() => getExpirationRules({ store, currentLanguage }))
```

Comment:
Why did you decide to create a helper instead of a composable if you're only ever using it in this `computed` here?

Reply:
Mainly to keep all link helpers in one place. IMO in the long term all these helpers should be one (or more) composable(s)... but it can't be done yet because of our pesky quick actions 😅 

But I could start creating a composable for this now and we can transition with time?

Reply:
Yep, that would be cool. Start the composable and basically just move this one `computed` from here into the composable.